docs/tools.md

```diff
@@ -186,6 +186,35 @@ Benchmark                               Time             CPU      Time Old
 This is a mix of the previous two modes, two (potentially different) benchmark binaries are run, and a different filter is applied to each one.
 As you can note, the values in `Time` and `CPU` columns are calculated as `(new - old) / |old|`.
 
+## Note: Interpreting the output
+
+When comparing benchmarks, `compare.py` uses statistical tests to determine whether the performance has significantly changed. 
```

Comment:
I think this should start with a more human-facing explanation first.
Something to the effect of:

```
Performance measurements are an art, and performance comparisons are doubly so.
Results are often noisy, and don't necessarily have large absolute difference between them,
so just by visual inspection, it is not at all apparent if two measurements are actually showing
a performance change or not. It is even more confusing with multiple benchmark repetitions.

Thankfully, what we can do, is use statistical tests on the results
to determine whether the performance has statistically-significantly changed.
`compare.py` uses [Mann–Whitney U test](https://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U_test),
with a null hypothesis being that there's no difference in performance.
```
and streamline the rest.